build.gradle.kts

```diff
@@ -46,6 +46,7 @@ kotlin {
 		getByName("commonTest") {
 			dependencies {
 				implementation(libs.assertk)
+				implementation(libs.coroutines.test)
```

Comment:
I would add coroutines-core and put it as a commonMain dependency with a shared version. There's no guarantee that you're getting the 1.8.1 version of coroutines core in main here (although it's likely since it's been out forever), and it definitely won't be so when 1.9.0 releases very soon. Renovate will make that bump, and you're likely to get linking problems when internal APIs change.